nms/app/packages/magmalte/app/state/lte/EquipmentState.js

```diff
@@ -134,7 +135,7 @@ export async function FetchEnodebs(props: FetchProps) {
         return newEnb;
       }
     } catch (e) {
-      return {[id]: {enb_state: {} || {}, enb: enb}};
+      return {[id]: {enb_state: {}, enb: enb}};
```

Comment:
You might have to fix two lines earlier too 
`const newEnb = {[id]: {enb_state: newEnbSt || {}, enb: enb}};`